packages/plugin-core/src/workspace.ts

```diff
@@ -351,6 +379,18 @@ export class DendronWorkspace {
     this.L.info({ ctx, msg: "initialized" });
   }
 
+  getOrThrowWorkspaceImpl(): DWorkspaceV2 {
```

Comment:
nitpick: to be consistent with the other get-or-throwers in the codebase

```suggestion
  getWorkspaceImplOrThrow(): DWorkspaceV2 {
```

Reply:
updated